Makefile

```diff
@@ -20,5 +22,4 @@ publish:
 
 .PHONY: update
 update:
-	npm --prefix es-vue-base run build
```

Comment:
Why the removal of this? It needs to happen at least the first time before you run `make dev`

Reply:
It breaks if npm packages aren't installed